### PR TITLE
few more fields added in search field

### PR DESCRIPTION
**Pull request by tareq89**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Redirect user from advanced search to advanced search result page
[#8933](https://github.com/opencrvs/opencrvs-core/issues/8933)

## Checklist

- [x] I have linked the correct Github issue under "Development"
- [x] I have tested the changes locally, and written appropriate tests
- [x] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [x] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: src/form/v2/birth/advancedSearch.ts [hunk 1]
```diff
@@ -20,11 +20,7 @@ export const advancedSearchBirth = [
       description: 'The title of Registration details accordion',
       id: 'v2.advancedSearch.form.registrationDetails'
     },
-    fields: [
-      {
-        fieldId: 'child.dob'
-      }
-    ]
+    fields: []
```
Comment (cibelius) [inline src/form/v2/birth/advancedSearch.ts:23]: Is this empty section left in here on purpose?

**Review by cibelius — APPROVED** [review]

File: src/form/v2/birth/advancedSearch.ts (hunk 1 above)
Comment (tareq89) [reply src/form/v2/birth/advancedSearch.ts:23]: Yes, currently develop only allows to search inside the declaration data, later, when searching inside metadata will be available, this will be filled up.
